client/src/modules/debtors/groups.service.js

```diff
@@ -117,5 +118,11 @@ function DebtorGroupService(Modal, Session, $translate, Api) {
       .then(service.util.unwrapHttpResponse);
   }
 
+  function history(debtorUuid, limit) {
+    const url = `${baseUrl}history/${debtorUuid}/${limit}`;
```

Comment:
Could you make the `limit` into a query string?  That way, it would be option and you could either see the entire history with `/history/:uuid` or a slice of the history with `/history/:uuid?limit=X`.